test/cli.js

```diff
@@ -0,0 +1,83 @@
+const childProcess = require('child_process');
+const fs = require('fs');
+const glob = require('fast-glob');
+const os = require('os');
+const path = require('path');
+const yargs = require('yargs');
+
+async function run(argv) {
+  const workspaceRoot = path.resolve(__dirname, '../');
+
+  const gitignore = fs.readFileSync(path.join(workspaceRoot, '.gitignore'), { encoding: 'utf-8' });
+  const ignore = gitignore.split(os.EOL).filter((pattern) => {
+    return pattern.length > 0 && !pattern.startsWith('#');
+  });
+  const globPattern = `**/*${argv.testFilePattern}*.test.{js,ts,tsx}`;
+  const spec = glob.sync(globPattern, {
+    cwd: workspaceRoot,
+    ignore,
+  });
```

Comment:
Had the fastest bootstrap performance for me. `node-glob` traverses all files matching the pattern first before applying `ignore` which means traversing all of `node_modules`. Took ~5s to find matched files

`globby` was also ok performance-wise and should have the most accurate results since it should also include nested .gitignores. If this becomes a concern we can switch to `globby`.

`fast-glob` took ~0.4s to find matched files.